wgkex/common/logger.py

```diff
@@ -0,0 +1,13 @@
+from logging import basicConfig
+from logging import DEBUG
+from logging import info as info
+from logging import warning as warning
+from logging import error as error
+from logging import critical as critical
+from logging import debug as debug
+
+basicConfig(
+    format="%(asctime)s,%(msecs)d %(levelname)-8s [%(filename)s:%(lineno)d] %(message)s",
+    datefmt="%Y-%m-%d:%H:%M:%S",
+    level=DEBUG,
```

Comment:
Before we had `level=config.load_config().get("log_level")` or similar.
Can we somehow use this or something similar here?

Reply:
If we import config module in the logging module, then we will end up with a circular dependency (config depends on logging, logging depends on config). 

If we remove logging from config module (Very possible, it's 2 lines), then all tests will break. This is because we import the modules before mocking, which means any call to load config will fail as they're unmocked. 

An option that is viable is to expand the logging section of the wgkex.yaml file as per https://docs.python.org/3/library/logging.config.html#dictionary-schema-details , and IF the file exists on disk, and IF logging section is configured, use this as logging configuration. 

Else, fail back to a pre-configured default. I'll prep this now.